tcms/testplans/templates/testplans/get.html

```diff
@@ -126,7 +126,27 @@ <h2 class="card-pf-title">
                             <span class="fa fa-ellipsis-v"></span>
                         </button>
                         <ul class="dropdown-menu dropdown-menu-right" aria-labelledby="dropdownKebabRight9">
-                            <li><a href="#">{% trans 'Change status' %}</a></li>
+                            <li class="dropdown-submenu dropdown-menu-left">
+                                <a tabindex="-1" href="#">{% trans 'Priority' %}</a>
+                                <ul class="dropdown-menu js-test-case-menu-priority">
+                                    <li class="list-group-item'">
+                            <a></a>
+                        </li>
+                                </ul>
+                            </li>
+                            <li class="dropdown-submenu dropdown-menu-left">
+                                <a tabindex="-1" href="#">{% trans 'Status' %}</a>
+                                <ul class="dropdown-menu js-test-case-menu-status">
+                                    <li class="list-group-item'">
+                            <a>qwe</a>
+                        </li><li class="list-group-item'">
+                            <a>qwe2</a>
+                        </li>
+                                </ul>
+                            </li>
+                            <li><a href="#" class="js-test-case-menu-edit">{% trans 'Edit' %}</a></li>
+                            <li><a href="#" class="js-test-case-menu-delete">{% trans 'Delete' %}</a></li>
+                            <li><a href="#" class="js-test-case-menu-tester">{% trans 'Default tester' %}</a></li>
```

Comment:
The '#' anchor has an annoying side effect. Whenever I delete something the content jumps at the top of the page. One possibility is to use an anchor like `#tc-123` but I don't know how well that will work for deletions, if at all.

Reply:
I believe that removing of the href attribute will do the trick.